Makefile

```diff
@@ -419,6 +423,7 @@ ci-rp:
 		-f Dockerfile.ci-rp \
 		--ulimit=nofile=4096:4096 \
 		--build-arg REGISTRY=${REGISTRY} \
+		--build-arg BUILDER_REGISTRY=${BUILDER_REGISTRY	} \
```

Comment:
Is there a reason that `ci-rp` uses _shell variables_ here (`${WHATEVER}`) whereas `ci-tunnel` uses _make variables_ (`$(WHATEVER)`)? The former is not set:

```
docker build . --load \
	-f Dockerfile.ci-rp \
	--ulimit=nofile=4096:4096 \
	--build-arg REGISTRY=registry.access.redhat.com \
	--build-arg BUILDER_REGISTRY= \
	--build-arg ARO_VERSION=pr-2461469249-708360bff4bd9e0093b74456962d842f0c65564d \
	--no-cache=true \
	-t arosvcdev.azurecr.io/aro:pr-2461469249-708360bff4bd9e0093b74456962d842f0c65564d
```

Reply:
cc @shubhadapaithankar for context here.

Reply:
Fixed in https://github.com/Azure/ARO-RP/pull/4195/commits/80c3ebadcb0754553d2ebcd049226453dedbd7ca